src/App.ts

```diff
@@ -1088,6 +1093,7 @@ export default class App<AppCustomContext extends StringIndexed = StringIndexed>
     if (type === IncomingEventType.Action && context.functionExecutionId !== undefined) {
       listenerArgs.complete = CustomFunction.createFunctionComplete(context, client);
       listenerArgs.fail = CustomFunction.createFunctionFail(context, client);
+      listenerArgs.inputs = context.functionInputs;
```

Comment:
Just sharing as an option, free to ignore- all listener arguments could be [created](https://github.com/slackapi/bolt-js/blob/54f36928ae8cb5cabc59e71bce4346d2c23ef019/src/CustomFunction.ts#L176) and assigned with `Object.assign` if we wanted to reduce the number of `CustomFunction` exports